aea/crypto/ethereum.py

```diff
@@ -365,6 +365,20 @@ def recover_message(
             )
         return (address,)
 
+    @classmethod
+    def recover_verifying_keys_from_message(
+        cls, message: bytes, signature: str, is_deprecated_mode: bool = False
+    ) -> Tuple[str, ...]:
+        """
+        Get the public key used to produce the `signature` of the `message`
+
+        :param message: raw bytes used to produce signature
+        :param signature: signature of the message
+        :param is_deprecated_mode: if the deprecated signing was used
+        :return: the recovered public keys
+        """
+        raise NotImplementedError
```

Comment:
Why this is not implemented? Maybe add a comment above the raise statement.

Reply:
@marcofavorito want to implement? It should not be hard. Just didn't have the time.